inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,82 @@
+package inheritance;
+
+// Thanks to James Dansie for providing help with the logic and some code
+
+// Restaurant gets a linked list of reviews.
+// The methods exist to update the constructor function, given a review added.
+
+public class Restaurant {
+    String name;
+    double stars0To5;
+    int priceCategory;
```

Comment:
it would be good for a restaurant to hold reviews that pertain to it, probably a list